doc/man/8/radosgw-admin.rst

```diff
@@ -424,9 +768,9 @@ Options
 
 	List of caps (e.g., "usage=read, write; user=read".
 
-.. option:: --compression=<compression-algorithm>
+.. option:: --compression=<type>
```

Comment:
I prefer the original text

Reply:
I was making it similar to the text in rgw_admin.cc. Anyway, reverted it.